core/livepeernode.go

```diff
@@ -94,25 +94,13 @@ func NewLivepeerNode(e eth.LivepeerEthClient, wd string, dbh *common.DB) (*Livep
 
 func (n *LivepeerNode) StartEthServices() error {
 	var err error
-	for k, s := range n.EthServices {
-		// Skip BlockService until the end
-		if k == "BlockService" {
-			continue
-		}
+	for _, s := range n.EthServices {
 		err = s.Start(context.Background())
```

Comment:
It would be much more efficient to declare `ctxt := context.Background()` before this range loop and pass `ctxt` into `Start()` instead, rather than fetching `context.Background()` each time.

Reply:
This code is related to the old event monitor and will eventually be removed once we refactor the two services, but you're totally right if we were to keep this!  Check out this comment: https://github.com/livepeer/go-livepeer/pull/1046#discussion_r317320941